File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Sat/ConsultarSAT.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Sat
{
    class ConsultarSAT : SatCommand
    {
        readonly int numSessao;

        public ConsultarSAT(int numSessao) : base("ConsultarSat")
        {
            this.numSessao = numSessao;
        }

        protected override string FunctionParameters()
        {
            return "\"numSessao\"" + ":" + numSessao;
        }
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/SAT/Commands/ConsultarSAT.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.SAT.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.SAT.SatCommand

class ConsultarSAT(private val numSessao: Int) : SatCommand("ConsultarSat") {
    override fun functionParameters(): String {
        return "\"numSessao\"" + ":" + numSessao
    }
}